Scripts/Services/LootGeneration/RunicReforging/RunicReforging.cs

```diff
@@ -1938,18 +1921,23 @@ public static void GenerateRandomItem(Item item, Mobile killer, int basebudget,
 
                     int divisor = GetDivisor(basebudget);
 
-                    double perc = 0.0;
-                    double highest = 0.0;
+                    int luckyRandom = 10000;
```

Comment:
Main part of the change:
before: the perc is calculated by constantly rerolling and replacing if its a higher value
now: the perc is rolled once, but then rerolled against the previous number for each 600. then any remaining luck is calculated in too
example: with 900 luck
1st roll as normal:                     luckyRandom =  random(1,10000)=3700
2nd roll for 600 luck :                luckyRandom = random(3700,10000)=6000
3rd roll for 300 remaining luck:  6000+ (random(6000,10000)-6000)*0.5  = 7000